### PR TITLE
add glossary

### PR DESCRIPTION
**Pull request by luk3b**
I'm beginning to add the glossary as planned in #4, work in progress!

### CODE REVIEW

**Comment by pbrinkmeier** [comment]
Am I supposed to proofread/review this already?

**Comment by luk3b** [comment]
If you want to, sure ;)

File: betriebssysteme/glossary.md [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
 
+**API**: Application programming interface
+
+**NIC**: Network interface card
+
+**MMU**: Memory management unit _(built-in CPU hardware that translates virtual to physical addresses)_
+
+**PCB**: Process control block _(contains information about allocated resources)_
+
+**AS**: Address space _(all virtual memory locations a program can name, consists of sections like stack, data and text)_
```
Comment (pbrinkmeier) [inline betriebssysteme/glossary.md]: Why did you highlight the text in parens?

Comment (luk3b) [reply betriebssysteme/glossary.md]: what?

Comment (pbrinkmeier) [reply betriebssysteme/glossary.md]: Why did you make the text in parentheses italics?

**Pull request merged**